src/open_gira/io.py

```diff
@@ -135,7 +138,7 @@ def netcdf_packing_parameters(minimum: float, maximum: float, n_bits: int) -> Tu
 
     # _FillValue used to representing NaN as serialised integer
     # we have kept room at the ends of the integer bit space to avoid a collision
-    fill_value = -2 ** (n_bits - 1)
+    fill_value = -(2 ** (n_bits - 1))
```

Comment:
Did black add the brackets?

Reply:
Yes: [blame](https://github.com/nismod/open-gira/blame/7eae18d2b959ba08d35c006287ba73e7b263257d/src/open_gira/io.py#L141)